3rd-party-tools/beagle/docker_versions.tsv

```diff
@@ -1,2 +1,3 @@
 us.gcr.io/broad-gotc-prod/imputation-beagle:1.0.0-17Dec24.224-1740423035
 us.gcr.io/broad-gotc-prod/imputation-beagle:1.1.0-17Dec24.224-1758207261
+us.gcr.io/broad-gotc-prod/imputation-beagle:1.0.1-17Dec24.224-1763664735
```

Comment:
we should change the way we tag these to using whatever branch or git tag we're pulling from, so this should probably be something like 
```suggestion
us.gcr.io/broad-gotc-prod/imputation-beagle:1.0.1-base-version-1763664735
```